### ISSUE TITLE
Treeview not extended beyond first tag

### ISSUE THREAD

**Issue by humanfactors** [joaoasrosa/nppxmltreeview#7]
I have a document structure that is quite lengthy. Other XML readers have no problem with it.

Below is what it reads up to. However, it fails to actually get the treeview of anything under config or in any of the other parts of atc:experiment. Any ideas?

```
<?xml version="1.0" encoding="UTF-8" standalone="no" ?>

<atc:experiment 
  xmlns:atc="http://www.humanfactors.uq.edu.au/atc/2006/atc-ns" 
  atc:idx="Experiment1" 
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
  xsi:schemaLocation="http://www.humanfactors.uq.edu.au/atc/2006/atc-ns atc_experiment.xsd">

    <atc:config>
    </atc:config>
```

**Comment by joaoasrosa** [comment]
Hi @middlepathway .

I'm preparing a new version of the plugin. Can you share the file with me, in order to test it?
If it's contains sensitive data, can you anonymize it?

Cheers, João Rosa

**Comment by humanfactors** [comment]
I've worked out what the issue is: it cannot handle comments. It treats a `<!-- Comment -->` as the end of the file for some reason. I've attached the same file with and without comments. Rename the files as required for xml.

Thanks heaps.

[NPP_nocomments.xml.txt](https://github.com/joaoasrosa/nppxmltreeview/files/193390/NPP_nocomments.xml.txt)
[NPP_comments.xml.txt](https://github.com/joaoasrosa/nppxmltreeview/files/193391/NPP_comments.xml.txt)

**Comment by joaoasrosa** [comment]
Thanks for it.
I will analyse and fix in the next version.

2016-03-29 13:14 GMT+01:00 Michael Wilson notifications@github.com:

> I've worked out what the issue is: it cannot handle comments. It treats a  as the end of the file for some reason. I've attached the
> same file with and without comments. Rename the files as required for xml.
> 
> Thanks heaps.
> 
> NPP_nocomments.xml.txt
> https://github.com/joaoasrosa/nppxmltreeview/files/193390/NPP_nocomments.xml.txt
> NPP_comments.xml.txt
> https://github.com/joaoasrosa/nppxmltreeview/files/193391/NPP_comments.xml.txt
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> https://github.com/joaoasrosa/nppxmltreeview/issues/7#issuecomment-202862427

## 

Cumprimentos, João Rosa

**Comment by joaoasrosa** [comment]
Hi,
The issue is fixed in the Dublin release (1.2.0.25).

Can I use your files in the unit tests?

Regards, João Rosa

**Comment by humanfactors** [comment]
Feel free to use my files :) Thank you for the fix
